app/controllers/search_controller.rb

```diff
@@ -16,8 +16,8 @@ def search
     page = params[:page] || 1
     per_page = params[:per_page] || ENV['PER_PAGE'] || 20
     @results = search_results(page, per_page)
-    @pageable_results = paginate_results(page, per_page)
     return redirect_to root_url unless @results
+    @pageable_results = paginate_results(page, per_page)
```

Comment:
🌮 That is definitely the correct order.

(that was supposed to be a thumbs-up but I prefer the typo)